src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -246,6 +246,17 @@
   <data name="AvoidUnsealedAttributesMessage" xml:space="preserve">
     <value>Avoid unsealed attributes</value>
   </data>
+  <data name="AvoidConstArraysTitle" xml:space="preserve">
+    <value>Avoid const arrays</value>
+  </data>
+  <data name="AvoidConstArraysDescription" xml:space="preserve">
+    <value>Having a constant array passed as an argument is not ideally performant. Extract constant arrays as 'static readonly'.</value>
+    <comment>{Locked="static", "readonly"}</comment>
```

Comment:
I think it doesn't work like that. The syntax for multiple words is `{Locked="FirstWord"}{Locked="SecondWord"}. But there is no need to separate them.

```suggestion
    <comment>{Locked="static readonly"}</comment>
```

Reply:
Ok, done. I'll be working on reformatting the specific node rather than the whole document:
https://github.com/dotnet/roslyn-analyzers/blob/6af52ad4d3067b9a0237cc5993033ba682266082/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/AvoidConstArrays.Fixer.cs#L102
That's the only other comment I did not address, except for the `s_` prefix for `static` fields. I believe that, while that is a .NET source code style, it doesn't seem right to enforce a snake case formatting for field extractions.